jbmc/unit/solvers/refinement/string_constraint_instantiation/instantiate_not_contains.cpp

```diff
@@ -215,7 +215,7 @@ SCENARIO("instantiate_not_contains",
         simplify(sc.premise, empty_ns);
         simplify(sc.s0, empty_ns);
         simplify(sc.s1, empty_ns);
-        witnesses[sc] = generator.fresh_symbol("w", t.witness_type());
+        witnesses.emplace(sc, generator.fresh_symbol("w", t.witness_type()));
         nc_axioms.push_back(sc);
```

Comment:
Not sure that the .emplace adds a lot here (exprt is very cheap to construct).

Reply:
The `.emplace` is necessary, because a `symbol_exprt` cannot be default constructed.